packages/ds-medicare-gov/src/components/Dialog/Dialog.stories.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import Dialog from './Dialog';
+import { Button } from '@cmsgov/design-system';
+
+export default {
+  title: 'Medicare/Dialog',
```

Comment:
What does this new story do?

Reply:
So the Medicare Dialog has a different icon for the close button than core & healthcare. This story is a direct copy/paste of the original Dialog except it uses the mgov Dialog (with defaults) instead of core.

I wasn't sure of another way to show this subtle difference in the core story.

Reply:
Gotcha. If we want the visual regression tests to capture that difference, shouldn't we have it show the dialog by default rather than using a triggering button? Otherwise, we should probably configure loki to ignore this one because it's a duplicate of the original (just a screenshot of a solid button)

Reply:
I tried showing just the dialog modal in the story without any Button or interaction needed. However, this caused some issues with loading the React code on the documentation site. It seems to be something about the Dialog on the "docs" tab of Storybook that was creating issues.
I spent some time trying different variations and none were working, so I'm opting to skip for now.